pkgs/by-name/mv/mvnd/package.nix

```
@@ -44,22 +43,22 @@ maven.buildMavenPackage rec {
   ];
 
   mvnDepsParameters = mvnParameters;
-  mvnParameters = lib.concatStringsSep " " [
-    "-Dmaven.buildNumber.skip=true" # skip build number generation; requires a git repository
-    "-Drat.skip=true" # skip license checks; they require manaul approval and should have already been run upstream
-    "-Dspotless.skip=true" # skip formatting checks
+  mvnParameters = lib.concatStringsSep " " (
+    [
+      "-Dmaven.buildNumber.skip=true" # skip build number generation; requires a git repository
+      "-Drat.skip=true" # skip license checks; they require manaul approval and should have already been run upstream
+      "-Dspotless.skip=true" # skip formatting checks
 
-    # skip tests that fail in the sandbox
-    "-pl"
-    "!integration-tests"
-    "-Dtest=!org.mvndaemon.mvnd.client.OsUtilsTest,!org.mvndaemon.mvnd.cache.impl.CacheFactoryTest"
-    "-Dsurefire.failIfNoSpecifiedTests=false"
+      # skip tests that fail in the sandbox
+      "-pl"
+      "!integration-tests"
+      "-Dtest=!org.mvndaemon.mvnd.client.OsUtilsTest,!org.mvndaemon.mvnd.cache.impl.CacheFactoryTest,!org.mvndaemon.mvnd.client.NoDaemonTest"
+      "-Dsurefire.failIfNoSpecifiedTests=false"
 
-    "-Pnative"
-    # propagate linker args required by the darwin build
-    # see `buildGraalvmNativeImage`
-    ''-Dgraalvm-native-static-opt="-H:-CheckToolchain $(export -p | sed -n 's/^declare -x \([^=]\+\)=.*$/ -E\1/p' | tr -d \\n)"''
-  ];
+      "-Pnative"
+    ]
+    ++ lib.optional stdenv.hostPlatform.isDarwin "-DbuildArgs=-H:-CheckToolchain" # see `buildGraalvmNativeImage`
```

```suggestion
    ++ lib.optionals stdenv.hostPlatform.isDarwin [
       # see `buildGraalvmNativeImage`
      "-DbuildArgs=-H:-CheckToolchain"
    ]
```
Prefer using `lib.optionals`.